server/routes/comparison/comparisonController.test.ts

```diff
@@ -60,6 +60,26 @@ describe('Comparison controller', () => {
       expect(req.session.shortlist).toStrictEqual([prisoner1Detail.prisonNumber])
     })
 
+    it('should return empty list if no prisoners to compare', async () => {
+      historicalPrisonerService.getPrisonerDetail.mockResolvedValue(prisoner1Detail)
+      req.session.shortlist = []
+
+      await controller.getComparisonDetail(req, res)
+
+      expect(historicalPrisonerService.getPrisonerDetail).not.toHaveBeenCalled()
+      expectRenderComparisonContainingDetail([])
+    })
+
+    it('should return empty list if >3 prisoners to compare', async () => {
```

Comment:
I think 1 prisoner to compare is ok - if the user is removing items from the shortlist